Explorer/Assets/DCL/SceneRestrictionBusController/SceneRestriction/ExperiencesBlockedRestriction.cs

```diff
@@ -0,0 +1,11 @@
+namespace DCL.SceneRestrictionBusController.SceneRestriction
+{
+    public class ExperiencesBlockedRestriction : SceneRestrictionBase
```

Comment:
As far as I understood it's a type of restriction that should be implemented in the future.

In the PR description I stated that there are 2 restriction types that aren't currently implemented because are not currently handled by E@:

1. Passports can not be opened
2. Experiences are blocked

but they should be implemented in the future.
I created the respective bus message for later usage `ExperiencesBlockedRestriction` and `PassportBlockedRestriction`.

Should I delete them anyway?

Reply:
Cool, if there is a short term plan, lets keep it